### ISSUE TITLE
[Bookmark]: administration

### ISSUE THREAD

**Issue by jbeaujour** [sni/Thruk#1203]
Hello
when I modify a bookmark, the list is changed. His name appears at the bottom of the list. the initial order is not maintained.
Is it possible to add for bookmarks:
- at a minimum: maintaining order when the name does not change
- better: order management, with the possibility of freezing certain bookmarks at the top of the list for example (header) and / or moving the bookmarks (according to a hidden internal order number)
- and finally to have a "modify" button on a bookmark
Thanks

**Comment by sni** [comment]
at least bookmarks can be reordered now. Editing is still not possible (except by overwriting an existing bookmark by creating a new one with the same name)

**Comment by jbeaujour** [comment]
Hello
thank you for considering this request.
indeed we have various internal or external teams working on THRUK with specific needs without always wanting to ask for specific menu management.
Will it only be on THRUK v3?
And if so, when do you expect to release the first stable version?
Thanks

**Comment by sni** [comment]
Yes, that is Thruk 3 only. The release is planned for in the next 2-3 weeks.
